src/_includes/core-nodes/sort-use-case.md

```diff
@@ -0,0 +1,50 @@
+## What is sort node in Node-RED?
+
+The Sort node in Node-RED is a function node used to sort either an array of data within a message or a sequence of messages themselves, either in ascending or descending order.
+
+## Configuring Sort node
+
+- **Sort:**
+    - `msg.<property>`: Specify the message property which contains array data and needs to be sorted.
+    - Message sequences: Sort a sequence of messages. selecting this sort node relies on the received messages to have `msg.parts` set. The split node generates this property, but it can be manually created. It has the following properties:
```

Comment:
```suggestion
    - Message sequences: Sort a sequence of messages. Selecting this sort node relies on the received messages to have `msg.parts` set. The split node generates this property, but it can be manually created. It has the following properties:
```